contrib_bots/run.py

```diff
@@ -6,19 +6,28 @@
 import optparse
 import os
 import sys
+import django
 
 our_dir = os.path.dirname(os.path.abspath(__file__))
 
 # For dev setups, we can find the API in the repo itself.
 if os.path.exists(os.path.join(our_dir, '../api/zulip')):
     sys.path.append('../api')
+if os.path.exists(os.path.join(our_dir, '../zerver')):
+    sys.path.append('../')
+
+os.environ['DJANGO_SETTINGS_MODULE'] = 'zproject.settings'
```

Comment:
We should assume that most bot authors have no direct access to Django.  So we need to find a better way to get the bot name, or, better yet, figure out a more direct way to figure out that the bot was mentioned.